snuba/snuba_migrations/uptime_monitor_checks/0001_uptime_monitor_checks.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import AddIndicesData, OperationTarget, SqlOperation
+
+storage_set = StorageSetKey.UPTIME_MONITOR_CHECKS
+table_prefix = "uptime_monitor_checks"
+local_table_name = f"{table_prefix}_local"
+dist_table_name = f"{table_prefix}_dist"
+
+## what about all the fancy codecs? do we need those?
+columns: List[Column[Modifiers]] = [
+    Column("organization_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("environment", String(Modifiers(nullable=True, low_cardinality=True))),
+    Column("uptime_subscription_id", UInt(64)),
+    Column("uptime_check_id", UUID()),
+    Column("scheduled_check_time", DateTime()),
+    Column("timestamp", DateTime()),
+    Column("_sort_timestamp", DateTime()),
+    Column("duration", UInt(64)),
+    Column("region_id", UInt(16, Modifiers(nullable=True))),
```

Comment:
maybe this should be region slug? or need to increase to UInt 64? TBD.

Reply:
Are we going to have that many regions? Perhaps a LowCardinality(String), not null (make it a `global` region instead of `null`) is better and with the automatic compression (`ZSTD(1)` by default), this will be a very small column.

Reply:
yeah, we're also discussing on the uptime team side if we want to just use a slug everywhere for the region. will update this.

Reply:
`Nullable` makes ClickHouse maintains 2 columns instead of 1 with one column just saying if it's null or not. It's a bit wasteful. If we can avoid it, we should.

Reply:
👍